.gitignore

```diff
@@ -2,8 +2,9 @@ node_modules
 .DS_Store
 
 # Build products
-component.js
-function.js
-index.js
-mixin.js
-shallowEqual.js
\ No newline at end of file
+/component.js
+/decorator.js
+/function.js
+/index.js
+/mixin.js
+/shallowEqual.js
```

Comment:
Otherwise it picks up the files under src/. I could `add -f`, but this seems better.